www/simple.php

```diff
@@ -411,6 +547,9 @@ function getScheme() {
 	header('HTTP/1.0 204 No Content');
 //	header('HTTP/1.0 200 OK'); // dulwich swift-repo does not understand 204
 	nocache();
+} else if($mode=='autoregister') {
```

Comment:
I already merged and tested your PR .. but I think there should be an "autoregister sucessfull"  message in this case.